### ISSUE TITLE
partial input point

### ISSUE THREAD

**Issue by shuaiqianende** [yyxssm/PointSSC#8]
![image](https://github.com/user-attachments/assets/675cf7ef-45b2-48c5-905e-e690ede15f24)
I downloaded the file "V2X-Seq-SPD-infrastructure-side-velodyne" from the link you provided, and after extracting it, I got the PCD input point cloud files. However, the visualization results seem to have no direct correspondence with the ground truth (GT). Is there any processing required? 

![image](https://github.com/user-attachments/assets/38cb674c-12c8-4376-b5d9-770843cb8f79)
It seems that the official website is currently down. Could you please directly send me the input point cloud link (via Baidu cloud storage or other methods)

**Comment by yyxssm** [comment]
Our GT completion point clouds are generated by our "Fig. 2: Pipeline of our PointSSC dataset generation." in our paper.

I haven't organized and uploaded these codes because they are written based on another code framework. I will organize these codes and release them later. If you need them, I can share the raw code in these days.

![fig2](https://github.com/user-attachments/assets/7c4f193a-5540-40e8-bd04-b0aad09d1951)

**Comment by yyxssm** [comment]
Of course. But I guess the website crash is only temporary. If it's still inaccessible in a few days, I can send the data to you directly.

**Comment by shuaiqianende** [comment]
> pletion point clouds are generated by our "Fig. 2: Pipeline of our PointSSC dataset generation." in our paper.
> 
> I haven't organized and uploaded these codes because they are written based on another code framework. I will organize these codes and release them later. If you need them, I can share the raw code in these days.
I mean that the input point cloud I downloaded doesn't seem to correspond to the ground truth (GT) point cloud. Did I download the wrong input point cloud, or is there additional processing required after downloading the input point cloud?

**Comment by yyxssm** [comment]
I believe the dataset you downloaded is correct. I cropped the range of the point cloud, keeping only point clouds within the field of view of the infrastructure-side camera. Therefore, the range of my GT point cloud is slightly smaller.

**Comment by shuaiqianende** [comment]
Could you please explain specifically how to crop to a given camera view, and what specific parameters are required?

**Comment by shuaiqianende** [comment]
> I believe the dataset you downloaded is correct. I cropped the range of the point cloud, keeping only point clouds within the field of view of the infrastructure-side camera. Therefore, the range of my GT point cloud is slightly smaller.

Without the input point cloud, I won't be able to proceed with my experiment. If it's convenient, could you please share a contact method? My QQ is: 3443888106. Thank you very much!

**Comment by yyxssm** [comment]
Thanks for you reminder, I will upload the processed point cloud dataset.

**Comment by yyxssm** [comment]
The dataset has been uploaded [here](https://pan.baidu.com/s/1RIuNygbS0V08UTEqGerTsw?pwd=379x) (code: 379x).
